src/api.js

```diff
@@ -10,8 +10,14 @@ export const getArticles = (query = "") => {
   });
 };
 
-export const getArticle = (query = "") => {
-  return ncNewsApi.get(`/articles${query}`).then((response) => {
-    return response.data.article;
-  });
+export const getArticleById = (id) => {
+  return ncNewsApi
+    .get(`/articles/${id}`)
+    .then((response) => {
+      return response.data.article;
+    })
+    .catch((error) => {
+      console.error("Error fetching article:", error);
+      throw error;
+    });
```

Comment:
Good job extracting this out - because you throw an error here if I go to articles/1000 I just get an error in the console and in your app it says it's loading constantly. Remove this catch and you can keep the catch in your useEffect to catch if something goes wrong and your error may work. 

To be fair, dealing with an error here is a later task anyway!